.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,20 @@
+name: "CodeQL"
+
+on: [pull_request]
+jobs:
+  lint:
+    name: CodeQL
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 2
```

Comment:
Why do `fetch-depth: 2` when you can do `fetch-depth: 1` ?

Reply:
Not sure, copied it from somewhere. Replacing with what we have in release.yml, without this twos.